recovery/elliptics_recovery/dc_server_send.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import os
+import errno
+import time
+
+from elliptics_recovery.utils.misc import dump_key_data, load_key_data_from_file
+
+import elliptics
+
+log = logging.getLogger()
+
```

Comment:
Please add documentation for these new classes and methods, it is really hard to follow this new recovery mechanism without understanding what new classes are needed for